### ISSUE TITLE
Update signup function definition in aurelia-auth.d.ts

### ISSUE THREAD

**Issue by stuartbale** [paulvanbladel/aurelia-auth#126]
The signup function on auth-service supports an 'overload', where if the first argument is an object, it uses that, otherwise it generates an object from the supplied arguments:

```
signup(displayName, email, password) {
    let signupUrl = this.auth.getSignupUrl();
    let content;
    if (typeof arguments[0] === 'object') {
      content = arguments[0];
    } else {
      content = {
        'displayName': displayName,
        'email': email,
        'password': password
      };
    }
```

Would you please consider updating the aurelia-auth.d.ts Typescript definition to include this overload?

S

**Comment by paulvanbladel** [comment]
Would it be possible to make a PR please?

**Comment by Boolos** [comment]
This should be resolved with [PR #148] if you'd like to close this issue.
